CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 2.6.1
+
+* Prevent lib from crashing when listen to blestatus status throws an error on Android.
```

Comment:
- "lib"? ("library")
- "bblestatus"?? (either use `in-line code` quatoation and refer to an existing type or whatever, or [preferably?] use natural language)